app/components/list-uploaded-assets/ListUploadedAssets.tsx

```
@@ -0,0 +1,86 @@
+import { Asset } from '@app/lib/types';
+import { Box } from '@chakra-ui/react';
+import { useEffect, useMemo, useState } from 'react';
+import { Account } from 'thirdweb/wallets';
+import UploadAsset from './UploadedAsset';
+
+type ListUploadedAssetsProps = {
+  activeAccount: Account;
+};
+
+export default function ListUploadedAssets(props: ListUploadedAssetsProps) {
+  const [assets, setAssets] = useState<Asset[] | {}>();
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)